src/common/utils/gpo.h

```diff
@@ -230,6 +231,11 @@ namespace powertoys_gpo {
         return getConfiguredValue(POLICY_CONFIGURE_ENABLED_TEXT_EXTRACTOR);
     }
 
+    inline gpo_rule_configured_t getConfiguredPastePlainEnabledValue()
+    {
+        return getConfiguredValue(POLICY_CONFIGURE_ENABLED_PASTE_PLAIN);
+    }
```

Comment:
Any way I can test this and make sure I did it right?

Reply:
@carlos-zamora
I think the only way to test it is by setting the registry key manually.